src/Relation/Nullary/Negation.agda

```diff
@@ -8,16 +8,15 @@
 
 module Relation.Nullary.Negation where
 
-open import Effect.Monad
+open import Effect.Monad using (RawMonad; mkRawMonad)
 open import Data.Bool.Base using (Bool; false; true; if_then_else_; not)
-open import Data.Empty
-open import Data.Product as Prod
+open import Data.Empty using (⊥-elim)
+open import Data.Product.Base as Prod using (_,_; Σ; Σ-syntax; ∃; curry; uncurry)
```

Comment:
Do you use the import of `Σ-syntax` here?
if not, then one way to advance your thesis about the imports of `Data.Product.Base` might be to move *both* definitions of the syntax for `Σ` and `∃` to `Data.Product`?

Reply:
The syntaxes are used in a few places already. I don't want to rewrite that code to
drop the syntaxes (which are trivial definitions that have no external dependencies)
just for the sake of it.

I think this PR in its current shape should be backwards compatible.